includes/gateways/paypal/includes/class-wcs-paypal-standard-switcher.php

```diff
@@ -155,9 +155,10 @@ public static function maybe_set_payment_method( $payment_processing_result, $or
 	 * @since 2.0.15
 	 */
 	public static function save_old_paypal_meta( $order_id, $posted ) {
+		$order = wc_get_order( $order_id );
 
-		if ( wcs_order_contains_switch( $order_id ) ) {
-			$subscriptions = wcs_get_subscriptions_for_order( $order_id, array( 'order_type' => 'switch' ) );
+		if ( wcs_is_order( $order ) && wcs_order_contains_switch( $order ) ) {
```

Comment:
I'd be tempted to change this condition to do a early return and remove a level of indentation from the method 
```suggestion
		if ( ! wcs_is_order( $order ) || ! wcs_order_contains_switch( $order ) ) {
		    return;
		}
```

Reply:
Fixed in 886238ca. Thanks for the suggestion!